addons/zeus/functions/fnc_removeSoundZeus.sqf

```diff
@@ -16,3 +16,5 @@ if (isNull _unit) exitWith {hint "You have to select a unit to remove sound from
 
 // broadcast
 [QEGVAR(sounds,removeSound), [_unit]] call CBA_fnc_serverEvent;
+
+// TODO: should this remove ALL added sounds?
```

Comment:
on that object, I would say yes. can always re-add sounds if needed. Alternative would be to make new UI that if more than 1 sound is present on object, then it shows an dialog with option to remove all sounds, to select which ones to remove. 

Either is fine with me.

Reply:
As-is, it currently removes the earliest-added sound, but _stops_ all sounds (whoops).
I can update it so that it removes and stops all sounds, or only removes and stops the earliest-added sound.
Dialog _could_ be useful in some cases, but is probably overkill - my instinct says the most common use-case will be to remove all sounds.

Reply:
Agreed. Remove all sounds is the way